src/pages/MyPage/History/HistoryTable.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import styled from 'styled-components';
+
+function HistoryTable({ filteredList }) {
+  return (
+    <HistoryTableWrapper>
+      <thead>
+        <th>투자일</th>
+        <th className="col2">상품명</th>
+        <th>등급</th>
+        <th>예상수익률</th>
+        <th>기간</th>
+        <th>투자금액</th>
+        <th>원금상환률</th>
+        <th>상환완료회차</th>
+        <th>상환상태</th>
+      </thead>
+      <tbody>
+        {filteredList.map((item, idx) => {
```

Comment:
```suggestion
        {filteredList.map(({date, interestRate, term ~~~}, idx) => {
```
매개변수자리에서 바로 구조분해 가능합니다!

Reply:
세상에... 감사합니다!